Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallDiffCommand.cs

```diff
@@ -594,67 +594,45 @@ private void AddDirectory(string dirName, CancellationToken cancellationToken)
             }
 
             _logger.LogDebug("Checking whether patching file content is necessary...");
```

Comment:
This log is not longer accurate.

Reply:
I think this log is not needed. I removed it